### PR TITLE
added changes

### PR DESCRIPTION
**Pull request by urikirsh**
Changelog for version 7.10.1

### CODE REVIEW

File: docs/prover/changelog/prover_changelog.md [hunk 1]
```diff
@@ -5,6 +5,41 @@ Prover Release Notes
...
+- [bugfix] Make builtin sanity rule also check auto-generated assertions such as unwinding loop unroll assertions.
+- [feat] `DISPATCHER(true)` summaries will hard fail on type checking if there is no method in the scene that should be dispatched to.
+- [feat] Adding a warning when a contract alias conflicts with a contract in the scene.
+- [feat] View functions of contracts *other* than the current contract are now also excluded when checking invariants.
+- [feat] It's now possible to compare strings and bytes natively in CVL.
+- [feat] The dispatcher list summary now will also inline the fallback handler when using the `Contract._` syntax.
+- [feat] Transient storage support in invariants.
+- [feat] It's now possible to also write CVL summaries that include structs.
+- [syntax check] The usage of the keyword `lastReverted` led to incorrect specifications in cases when the last call was not using the `@withrevert` syntax. During type checking, the CVL parser will now fail and report an error.
+- [feature] The dispatcher summary `DISPATCHER(false|true)` has been enhanced to also include fallbacks.
```
Comment (johspaeth) [inline docs/prover/changelog/prover_changelog.md:20]: Minor nit - we use `[feat]` and from this line below `[feature]`
```suggestion
- [feat] The dispatcher summary `DISPATCHER(false|true)` has been enhanced to also include fallbacks.
```

**Review by johspaeth — COMMENTED** [review]
LGTM

File: docs/prover/changelog/prover_changelog.md [hunk 2]
```diff
@@ -5,6 +5,41 @@ Prover Release Notes
...
+- [feature] Model values for CVL-declared strings are now shown in the Variables Tab of the report.
+- [feature] Sanity rules are now explicitly shown as independent nodes in the rule view.
+- [feature] The browser tab icon in the rule report now displays the job execution status of a run: blue for running, green when the job has successfully been completed, and red when the job is halted or ends in an error state.
+- [feature] Improved display of ghost variable reads in the call trace.
+
+### Performance
+- The `-prover_args` option `-smt_easy_LIA` is now set to `true` by default.
+
+### CLI
+- [feature] The `—method` flag now also accepts a list of methods
```
Comment (jar-ben) [inline docs/prover/changelog/prover_changelog.md:36]: missing `.`

**Review by jar-ben — APPROVED** [review]
Looks reasonable

**Pull request merged**